src/tools/compiletest/src/header.rs

```diff
@@ -102,6 +102,9 @@ pub struct TestProps {
     pub dont_check_compiler_stdout: bool,
     // For UI tests, allows compiler to generate arbitrary output to stderr
     pub dont_check_compiler_stderr: bool,
+    // When checking the output of stdout or stderr check
+    // that the lines of expected output are a subset of the actual output.
+    pub compare_output_by_lines: bool,
```

Comment:
Can we rename this to include subset? Otherwise it's confusing since comparing output by lines is essentially what we *always* do...